Tools/Scripts/run-jsc-stress-tests

```diff
@@ -1852,8 +1852,8 @@ def runV8WebAssemblySuite(*optionalTestSpecificOptions)
         run("wasm-eager-jettison", "--forceCodeBlockToJettisonDueToOldAge=true", "--useRandomizingExecutableIslandAllocation=true", "--verifyGC=true", *(FTL_OPTIONS + optionalTestSpecificOptions))
         run("wasm-slow-memory", "--useWebAssemblyFastMemory=false", *(FTL_OPTIONS + optionalTestSpecificOptions))
         run("wasm-collect-continuously", "--collectContinuously=true", "--verifyGC=true", *(FTL_OPTIONS + optionalTestSpecificOptions)) if shouldCollectContinuously?
+        run("wasm-bbq", "--useWasmLLInt=false", *(FTL_OPTIONS + optionalTestSpecificOptions)) # Doesn't actually depend on FTL, but we enable it on the same platforms for now.
```

Comment:
I don't think the comment applies anymore, we're enabling it regardless of $isFTLPlatform.

I don't think we want to pass FTL_OPTIONS here if $isFTLPlatform is false. While `--useFTLJIT=true` is probably is a no-op, it's a bit strange to pass it if FTL isn't available on the target platform.